renderdoc/os/posix/apple/apple_stringio.cpp

```diff
@@ -326,6 +326,8 @@ void WriteOutput(int channel, const char *str)
     fprintf(stdout, "%s", str);
   else if(channel == OSUtility::Output_StdErr)
     fprintf(stderr, "%s", str);
+  else if(channel == OSUtility::Output_DebugMon && OSUtility::DebuggerPresent())
+    fprintf(stdout, "%s", str);
```

Comment:
As mentioned in the previous comment, this must still be gated so that it only outputs during replay, you cannot output to stdout during capture as the application may be using it.